content/rc/cloud-integrations/prometheus-integration.md

```diff
@@ -0,0 +1,169 @@
+                image: grafana/grafana
+                ports:
+                    - 3000:3000
+                environment:
+                    - GF_SECURITY_ADMIN_PASSWORD=secret
+                links:
+                    - prometheus-server:prometheus
+        ```
+
+    1. To start the containers, run:
+
+        ```sh
+        $ docker compose up -d
+        ```
+
+    1. To check that all the containers are up, run: `docker ps`
+    1. In your browser, sign in to Prometheus at http://<instance_ip>:9090 to make sure the server is running.
+    1. Select **Status** and then **Targets** to check that Prometheus is collecting data from the Redis Enterprise cluster.
+
+        {{<image filename="images/rs/prometheus-target.png" alt="The Redis Enterprise target showing that Prometheus is connected to the Redis Enterprise Cluster.">}}{{< /image >}}
+
+        If Prometheus is connected to the cluster, you can type **node_up** in the Expression field on the Prometheus home page to see the cluster metrics.
+
+1. Configure the Grafana datasource: 
+    1. Sign in to Grafana. If you installed Grafana with Docker, go to http://<instance_ip>:3000 and sign in with:
+
+        - Username: admin
+        - Password: secret
+
+    1. In the Grafana configuration menu, select **Data Sources**.
+
+    1. Select **Add data source**.
+
+    1. Select **Prometheus** from the list of data source types.
+
+        {{<image filename="images/rs/prometheus-datasource.png" alt="The Prometheus data source in the list of data sources on Grafana.">}}{{< /image >}}
+
+    1. Enter the Prometheus information:
+
+        - Name: `redis-enterprise`
+        - URL: `http://<your prometheus address>:9090`
+        - Access: `Server`
+
+        {{<image filename="images/rs/prometheus-connection.png" alt="The Prometheus connection form in Grafana.">}}{{< /image >}}
+
+    {{< note >}}
+
+- If the network port is not accessible to the Grafana server, select the **Browser** option from the Access menu.
+- In a testing environment, you can select **Skip TLS verification**.
+
+    {{< /note >}}
+
+1. Add dashboards for cluster, node, and database metrics.
+    To add preconfigured dashboards:
+    1. In the Grafana dashboards menu, select **Manage**.
+    1. Click **Import**.
+    1. Copy one of the configurations into the **Paste JSON** field.
+        {{% expand "database.json" %}}
+
+```json
+{{% embed-code "/rs/database.json" %}}
+```
+
+        {{% /expand%}}
+        {{%expand "node.json" %}}
+
+```json
+{{% embed-code "/rs/node.json" %}}
+```
+
+        {{% /expand%}}
+        {{%expand "cluster.json" %}}
+
+```json
+{{% embed-code "/rs/cluster.json" %}}
+```
+
+        {{% /expand%}}
+    1. In the Import options, select the `redis-enterprise` datasource and click **Import**.
+
+The dashboards that you create from the configurations are sample dashboards.
+For more information about configuring dashboards, see the [Grafana documentation](http://docs.grafana.org).
```

Comment:
It seems like this link redirects to another URL. Since this looks like an old link that uses http instead of https, maybe it's worth updating to the new link?